src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs

```diff
@@ -163,5 +164,50 @@ public void StringLastIndexOfChar(int size)
             }
             Assert.Equal(size / 2, index);
         }
+
+        private static string GenerateInputString(char source, int count, char replaceChar, int replacePos)
+        {
+            char[] str = new char[count];
```

Comment:
you could just use StringBuilder here - no strong opinion though

Reply:
this func is the duplicate I mentioned above

Reply:
There's no need for a StringBuilder here.